### ISSUE TITLE
named parameters

### ISSUE THREAD

**Issue by tsloughter** [artemeff/eql#5]
Was wondering if you were interested in named parameters like yesql?

I figured I'd take the yesql bnf and use that to do the parsing and create a list like for sql:

```sql
SELECT *
FROM users
WHERE country_code = :country_code
```

The query would be stored as:

```erlang
["SELECT * FROM users WHERE country_code = ", country_code]
```

Then extending `get_query` to take the named args:

```erlang
> eql:get_query(users_by_country, [{country_code, "US"}]). %% or #{country_code => "US"}
["SELECT * FROM users WHERE country_code = ", "US"]
```

Leaving it as an iolist.

**Comment by artemeff** [comment]
Useful functionality, with this feature we also can make composable queries. I'll be glad to see it :)

**Comment by artemeff** [comment]
Solved in #6
